src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Tests\Helper\Image;
+
+use Kunstmaan\MediaBundle\Entity\Media;
+use Kunstmaan\MediaBundle\Helper\Image\ImageHandler;
+use Kunstmaan\UtilitiesBundle\Helper\Slugifier;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
+use Kunstmaan\MediaBundle\Helper\MimeTypeGuesserFactoryInterface;
+use Kunstmaan\MediaBundle\Helper\ExtensionGuesserFactoryInterface;
+
+class ImageHandlerTest extends TestCase
+{
+    public function testPrepareWithSvg()
+    {
+        $mockMimeTypeGuesserfactory = $this->createMock(MimeTypeGuesserFactoryInterface::class);
```

Comment:
Do you really need a mock here?

```php
$factory = new MimeTypeGuesserFactory();
$handler = new ImageHandler(1, $factory,.....);
```
should be enuogh

`get()` will [return a new instance](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/master/src/Kunstmaan/MediaBundle/Helper/MimeTypeGuesserFactory.php) anyway 🤔

Reply:
Yes indeed, fixed!